lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.10.0-dev",
+  "version": "3.10.0-dev.1",
```

Comment:
We had to bump the dev version for all packages because `3.10.0-dev` was already published to npm. Otherwise, you could run into an issue when generating a project locally with create-mobify-app-dev.js script and failing to see the changes in the generated project.